### ISSUE TITLE
[ERROR] - error when reload the xib file

### ISSUE THREAD

**Issue by Carmudii** [johnno1962/InjectionIII#415]
Error when reloading the xib file, but when I'm reloading the viewController this work well.
what am I doing wrong?

<img width="920" alt="Screen Shot 2022-09-13 at 17 44 05" src="https://user-images.githubusercontent.com/64249071/189882699-0bcb1fb2-00ae-4385-8b04-6b74c68a3774.png">

```
Thread 1: Fatal error: 'try!' expression unexpectedly raised an error: Error Domain=NSCocoaErrorDomain Code=260 "The file “eval103.sh” couldn’t be opened because there is no such file." UserInfo={NSFilePath=/Users/carmudi/Library/Developer/CoreSimulator/Devices/452DE9AA-73F0-4EEF-9F08-5847D97A69F8/data/Containers/Data/Application/BBBA8816-A245-4CEE-B91D-2C2D262E06C7/tmp/eval103.sh, NSUnderlyingError=0x600000cdeac0 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
```

edit: I have the latest version

**Comment by johnno1962** [comment]
Hi, what version of macOS/Xcode/iOS is this? xib injection is at the very limit of being supported. It has worked in the past but it is no longer being maintained. This seems to be a new permission problem we haven't got to the bottom of yet that first emerged here: https://github.com/johnno1962/HotReloading/issues/59#issuecomment-1235404418. The file doesn't exist because it has not been saved successfully. Is this a downloaded GitHub release or the App Store version of the app?

**Comment by Carmudii** [comment]
macOS: BigSur 11.6
Xcode: 13.2.1 (13C100)
iOS emulator: 13.0

I've downloaded it from the GitHub release. I have also tried to give full access but the problem is still the same

**Comment by johnno1962** [comment]
Is code injection working?

**Comment by Carmudii** [comment]
code injection works fine without a problem. but when I'm editing a xib file the problem appears

**Comment by johnno1962** [comment]
I'm unable to replicate this problem on my machine. Any chance you could try with version 4.3.2 from github?

**Comment by Carmudii** [comment]
solved with 4.3.2 version thank you. I think I can close this ticket now

**Comment by johnno1962** [comment]
I've spent more time looking at this and it seems likely switching to 4.3.2 worked as there is a problem with the App Store (sandboxed) version of injection when you are running a simulator < iOS 14 (due to previous simulators only containing Intel architecture.) You should find downloading the 4.4.1 release from GitHub works with your project.
